log/log.go

```diff
@@ -49,7 +49,7 @@ func Debug(args ...interface{}) {
 }
 
 func init()  {
-    usr, err := user.Current()
+   /* usr, err := user.Current()
```

Comment:
why do you comment these statements?

Reply:
travis will fail whit test when create file, just comments them for quick pass

Reply:
travis will fail whit test when create file, just comments them for quick pass